README.md

```diff
@@ -16,7 +16,7 @@ for your development platform
 
 * Start the Function app - `WEBSITE_HOSTNAME=localhost:7071 func start`.
   Setting `WEBSITE_HOSTNAME` prevents an SSL error being reported when running
-  durable functions locally.
+  [durable functions locally](https://github.com/Azure/azure-functions-durable-js/issues/28#issuecomment-453250279).
```

Comment:
Just looked at the issue in github and it seems it was fixed in 1.1.3 which is now the latest release (as of 4 days ago).

Upgrading durable-functions should make the workaround redundant.

Reply:
Ok. Separate PR for that change?